File: flutter_bank_app_docker/bank_app/src/trading/trading.entity.ts
import { Entity, PrimaryGeneratedColumn, Column, CreateDateColumn } from 'typeorm';

@Entity('trading_data')
export class TradingEntity {
  @PrimaryGeneratedColumn('increment', { name: 'idtrading' })
  idtrading: number;

  @Column()
  type: string;

  @Column()
  name: string;

  @Column()
  symbol: string;

  @Column('decimal', { precision: 10, scale: 2})
  price: number;

  @CreateDateColumn()
  recordedAt: Date;
}

File: flutter_bank_app/lib/Domain/Entities/trading_entity.dart
class TradingEntity {
  final int? id;
  final String? type;
  final String? name;
  final String? symbol;
  final double? price;
  final DateTime? recordedAt;

  TradingEntity(
      {this.id,
      this.type,
      this.name,
      this.symbol,
      this.price,
      this.recordedAt});
}
